js/main.js

```diff
@@ -0,0 +1,8 @@
+// const ele1 = document.querySelector(".span2");
+// const ele2 = document.querySelector("button");
+// const ele3 = document.querySelector("span");
+
+// eslint-disable-next-line no-console
+console.log('first hello world');
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Just as your previous reviewer pointed out, kindly include a functionality code for the Mobile menu 👍 

-  [ ] [ Optional ] It is not a good practice to include a `console.log` object on the code base. on line `6`, I suggest you take out the `console.log('first hello world');` command. That will help you get used to writing efficient and industry standard code 😃 

- [ ]  [ Optional ] Do well not to disable linters in your code. It's included in your code base to help you keep to best practices. I suggest you remove it in lines 5` and `7`